demos/go/gin/main.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	lr "github.com/LoginRadius/go-sdk"
+	account "github.com/LoginRadius/go-sdk/api/account"
+	lrauthentication "github.com/LoginRadius/go-sdk/api/authentication"
+	"github.com/LoginRadius/go-sdk/lrerror"
+	"github.com/gin-gonic/contrib/static"
+	"github.com/gin-gonic/gin"
+	"github.com/joho/godotenv"
+)
+
+var router *gin.Engine
+
+func main() {
+	err := godotenv.Load("config.env")
+
+	if err != nil {
+		log.Fatal("Error loading env files")
+	}
+	// Set the router as the default one shipped with Gin
+	router := gin.Default()
+
+	// Serve frontend static files
+	router.Use(static.Serve("/", static.LocalFile("../../../theme", true)))
+
+	// Setup route group for the API
+	api := router.Group("/api")
+	{
+		api.GET("/", func(c *gin.Context) {
+			c.JSON(http.StatusOK, gin.H{
+				"message": "Successfully started",
+			})
+		})
+	}
+	api.GET("/profile", handleget)
+	api.POST("/profile", handlepost)
+	// Start and run the server
+	router.Run(":3000")
+
+}
+func handleget(c *gin.Context) {
+	var w http.ResponseWriter = c.Writer
+	var errors string
+	respCode := 200
+
+	cfg := lr.Config{
+		ApiKey:    os.Getenv("APIKEY"),
+		ApiSecret: os.Getenv("APISECRET"),
+	}
+	lrclient, err := lr.NewLoginradius(
+		&cfg,
+		map[string]string{"token": os.Getenv("ACCESS_TOKEN")},
```

Comment:
@gagan3012 
Get the token value from query param i.e `keys := c.Request.URL.Query()`

Reply:
@pareek-narendra fixed both requested changes